CONTRIBUTING.md

```diff
@@ -33,7 +67,7 @@ requests.
 
 The developer
 [mailing list](https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-dev)
-should be used to discuss complicated or controversial changes before working on
+should be used to discuss complicated or controversial consensus or protocol changes before working on
```

Comment:
maybe `P2P protoocol`? (to distinguish it from internal or interprocess interfaces)

Reply:
Done. Fixed in commit 100000d1